sys/net/wot/coap/coap.c

```diff
@@ -305,3 +398,148 @@ void wot_td_coap_server_init(void)
+    );
+}
+
+int wot_td_coap_event_enable(const char *key){
+    WOT_TD_COAP_ENABLE_x(
+        _wot_td_coap_event_resources, wot_td_coap_event_affordance_t,
+        wot_td_coap_find_event, wot_td_thing_event_add, event_affordance
+    );
+}
+
+int wot_td_coap_prop_toggle(const char *key){
+    WOT_TD_COAP_TOGGLE_x(
+        _wot_td_coap_prop_resources, wot_td_coap_prop_affordance_t, prop_affordance,
+        wot_td_coap_find_prop, wot_td_thing_prop_rm, wot_td_thing_prop_add);
+}
+
+int wot_td_coap_action_toggle(const char *key){
+    WOT_TD_COAP_TOGGLE_x(
+        _wot_td_coap_action_resources, wot_td_coap_action_affordance_t, action_affordance,
+        wot_td_coap_find_action, wot_td_thing_action_rm, wot_td_thing_action_add);
+}
+
+int wot_td_coap_event_toggle(const char *key){
+    WOT_TD_COAP_TOGGLE_x(
+        _wot_td_coap_event_resources, wot_td_coap_event_affordance_t, event_affordance,
+        wot_td_coap_find_event, wot_td_thing_event_rm, wot_td_thing_event_add);
+}
+
+int wot_td_coap_prop_affordance_disable(wot_td_coap_prop_affordance_t *affordance){
+    WOT_TD_COAP_DISABLE_AFF_x(_wot_td_coap_prop_resources, prop_affordance, wot_td_thing_prop_rm);
+}
+
+int wot_td_coap_action_affordance_disable(wot_td_coap_action_affordance_t *affordance){
+    WOT_TD_COAP_DISABLE_AFF_x(_wot_td_coap_action_resources, action_affordance,
+                              wot_td_thing_action_rm);
+}
+
+int wot_td_coap_event_affordance_disable(wot_td_coap_event_affordance_t *affordance){
+    WOT_TD_COAP_DISABLE_AFF_x(_wot_td_coap_event_resources, event_affordance,
+                              wot_td_thing_event_rm);
+}
+
+int wot_td_coap_prop_affordance_enable(wot_td_coap_prop_affordance_t *affordance){
+    WOT_TD_COAP_ENABLE_AFF_x(_wot_td_coap_prop_resources, prop_affordance, wot_td_thing_prop_add);
+}
+
+int wot_td_coap_action_affordance_enable(wot_td_coap_action_affordance_t *affordance){
+    WOT_TD_COAP_ENABLE_AFF_x(_wot_td_coap_action_resources, action_affordance,
+                             wot_td_thing_action_add);
+}
+
+int wot_td_coap_event_affordance_enable(wot_td_coap_event_affordance_t *affordance){
+    WOT_TD_COAP_ENABLE_AFF_x(_wot_td_coap_event_resources, event_affordance,
+                             wot_td_thing_event_add);
+}
+
+int wot_td_coap_prop_affordance_toggle(wot_td_coap_prop_affordance_t *affordance){
+    WOT_TD_COAP_TOGGLE_AFF_x(
+        _wot_td_coap_prop_resources, prop_affordance, wot_td_thing_prop_rm, wot_td_thing_prop_add);
+}
+
+int wot_td_coap_action_affordance_toggle(wot_td_coap_action_affordance_t *affordance){
+    WOT_TD_COAP_TOGGLE_AFF_x(
+        _wot_td_coap_action_resources, action_affordance, wot_td_thing_action_rm,
+        wot_td_thing_action_add);
+}
+
+int wot_td_coap_event_affordance_toggle(wot_td_coap_event_affordance_t *affordance){
+    WOT_TD_COAP_TOGGLE_AFF_x(
+        _wot_td_coap_event_resources, event_affordance, wot_td_thing_event_rm, 
```

Comment:
I introduced some trailing whitespace by mistake. (By the way, I guess the review commits could be squashed before merging.)

```suggestion
        _wot_td_coap_event_resources, event_affordance, wot_td_thing_event_rm,
```